app/src/main/kotlin/com/wire/android/ui/common/WireDropDown.kt

```diff
@@ -0,0 +1,157 @@
+        )
+        Column(
+            modifier = Modifier.clip(shape).background(color = MaterialTheme.wireColorScheme.secondaryButtonEnabled, shape = shape)
+                .border(width = 1.dp, color = borderColor, shape)
+                .clickable {
+                    expanded = !expanded
+                }) {
+            Row(Modifier.padding(start = 16.dp, end = 16.dp, top = 12.dp, bottom = 12.dp)) {
+                Column(Modifier.weight(1f).align(Alignment.CenterVertically)) {
+                    Text(
+                        text = if (selectedIndex != -1) {
+                            items[selectedIndex] + GetDefaultTextIndicator(selectedIndex, defaultItemIndex)
+                        } else placeHolderText,
+                        style = MaterialTheme.wireTypography.body01,
+                        color = if (selectedIndex != -1) MaterialTheme.wireColorScheme.onSecondaryButtonEnabled
+                        else MaterialTheme.wireColorScheme.secondaryText
+                    )
+                }
+                Icon(
+                    imageVector = Icons.Filled.ExpandMore,
+                    contentDescription = stringResource(R.string.change),
+                    tint = MaterialTheme.wireColorScheme.secondaryText,
+                    modifier = Modifier.padding(top = 4.dp).rotate(arrowRotation)
+                )
+            }
+            if (expanded) Column {
+                AnimatedVisibility(
+                    visible = expanded, enter = fadeIn(animationSpec = tween(500, 200)) + expandVertically(
+                        animationSpec = tween(
+                            500
+                        )
+                    ), exit = fadeOut(animationSpec = tween(400)) + shrinkVertically(
+                        animationSpec = tween(
+                            400
+                        )
+                    )
+
+                ) {
+                    Column {
+                        Divider()
+                        List(items.size) { index ->
+                            DropdownItem(text = items[index] + GetDefaultTextIndicator(index, defaultItemIndex),
+                                isSelected = index == selectedIndex,
+                                onClick = {
+                                    selectedIndex = index
+                                    expanded = false
+                                    onChange(selectedIndex)
+                                })
+                            if (index != items.size - 1) Divider()
+                        }
+                    }
+                }
+            }
+        }
+
+    }
+}
+
+@Composable
+private fun Divider() = Spacer(
+    modifier = Modifier.fillMaxWidth().height(1.dp).background(
+        MaterialTheme.wireColorScheme.secondaryButtonEnabledOutline
+    )
+)
+
+@Composable
+private fun GetDefaultTextIndicator(index: Int, defaultIndex: Int): String {
```

Comment:
I think we agreed not to add "get" prefixes to functions, also it's a `@Composable` but it returns `String` and doesn't actually draw anything, so I propose to change it to:
```
private fun Context.defaultTextIndicator(index: Int, defaultIndex: Int): String {
    val defaultText = getString(R.string.wire_dropdown_default_indicator)
    return if (index == defaultIndex) defaultText else String.EMPTY
}
```
and just call it this way:
`LocalContext.current.defaultTextIndicator(index, defaultItemIndex)`